core/echo.go

```diff
@@ -99,3 +101,26 @@ func getGroup(path string) string {
 	}
 	return ""
 }
+
+func creatorFn() EchoServer {
+	echoServer := echo.New()
+	echoServer.HideBanner = true
+	echoServer.Use(middleware.Logger())
+	echoServer.HTTPErrorHandler = httpErrorHandler
+	// TODO: We might not want to enable CORS on all endpoints (https://github.com/nuts-foundation/nuts-node/issues/60)
+	echoServer.Use(middleware.CORS())
+	return echoServer
+}
+
+func httpErrorHandler(err error, ctx echo.Context) {
+	if prb, ok := err.(*problem.Problem); ok {
+		// todo: connect to module logger
+		if !ctx.Response().Committed {
```

Comment:
what if the response already _is_ committed?

Reply:
there is nothing we can do if the response is already committed (besides logging). The current approach replicates echo's `DefaultHTTPErrorHandler()`